composer.json

```diff
@@ -73,7 +73,7 @@
         "sylius-labs/doctrine-migrations-extra-bundle": "^0.1.4 || ^0.2",
         "sylius-labs/polyfill-symfony-event-dispatcher": "^1.0.1",
         "sylius-labs/polyfill-symfony-framework-bundle": "^1.0 || ^1.1",
-        "sylius-labs/polyfill-symfony-security": "^1.1",
+        "sylius-labs/polyfill-symfony-security": "dev-fix/user-salt-interface",
```

Comment:
Waits for https://github.com/SyliusLabs/PolyfillSymfonySecurity/pull/8 to be merged.